templates/solutions/telco/_form-telco-5g-core.html

```diff
@@ -0,0 +1,94 @@
+<div class="p-modal" style="display: none;" id="telco-5g-core-modal">
```

Comment:
These fields are ordered a bit differently on the [doc](https://docs.google.com/document/d/1mi6NOW_GiCMZ1O4xRCxS2RPhZMjEaOdipD5LFnCvMz4/edit), could you have a look?

Reply:
I've rearranged them but kept "Phone number" on the bottom-most field. The `intl-tel-input` is removing bottom padding for the phone number so I will leave it on the bottom for now.